formats/organisation/frontend/examples/bona_vacantia.json

```diff
@@ -0,0 +1,36 @@
+{
+  "base_path": "/government/organisations/bona-vacantia",
+  "content_id": "c9109663-0128-4db4-b513-a9cdfb691034",
+  "title": "Bona Vacantia",
+  "format": "organisation",
+  "document_type": "organisation",
```

Comment:
For reference, as discussed in person, this should be `placeholder_organisation`